readme.md

```diff
@@ -1,1109 +1,61 @@
-
-```Java
-service.autodiscoverUrl("<your_email_address>");
-```
-
-We recommend that you use the Autodiscover service, for the following reasons:
-
-- Autodiscover determines the best endpoint for a given user (the endpoint that is closest to the user’s Mailbox server).
-- The EWS URL might change as your administrators deploy new Client Access servers.
-
-You can safely cache the URL that is returned by the Autodiscover service and reuse it. Autodiscover should be called periodically, or when EWS connectivity to a given URL is lost.
-Note that you should either set the URL manually or call AutodiscoverUrl, but you should not do both.
-
-## Responding to Autodiscover Redirecting
-
-If the domain that the user inputs as their email address contains a CNAME that redirects the user this Exception is thrown:
-
-> microsoft.exchange.webservices.data.autodiscover.exception.AutodiscoverLocalException: Autodiscover blocked a
-potentially insecure redirection to **URL**. To allow Autodiscover to follow the redirection, use the AutodiscoverUrl(string, AutodiscoverRedirectionUrlValidationCallback) overload.<
-
-When this happens, instead of failing, the user can be prompted to accept the redirection or
-not. That functionality needs to be implemented inside the autodiscoverRedirectionUrlValidationCallback method. In the example below, it only checks to see that the redirection url starts with "https://". To accomplish this
-```Java
-static class RedirectionUrlCallback implements IAutodiscoverRedirectionUrl {
-        public boolean autodiscoverRedirectionUrlValidationCallback(
-                String redirectionUrl) {
-            return redirectionUrl.toLowerCase().startsWith("https://");
-        }
+# EWS JAVA API
+
+Please see the [Getting Started Guide](https://github.com/OfficeDev/ews-java-api/wiki/Getting-Started-Guide) on our wiki for an introduction to this library.
+
+## Using the library
+Prebuilt JARs are available in the Maven Central repository, which are easy to use with your project. Note that currently, no stable version is available yet, only snapshots in the snapshots repository.
+
+### Maven
+If you want to use a snapshot build, add the Maven Central snapshots repository to your project's `pom.xml`. If you want to use a stable build (not available yet), you should skip this step.
+```xml
+<project>
+  <repositories>
+    <repository>
+      <id>sonatype-snapshots</id>
+      <name>Sonatype OSS Snapshots</name>
+      <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
+      <releases>
+        <enabled>false</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+    </repository>
+  </repositories>
+</project>
+```
+
+And finally, add the dependency to your project's `pom.xml`.
+```xml
+<project>
+  <dependencies>
+    <dependency>
+      <groupId>com.microsoft.ews-java-api</groupId>
+      <artifactId>ews-java-api</artifactId>
+      <version>2.0-SNAPSHOT</version>
```

Comment:
However, if we'd keep the version number here, this would be the easiest way to determine the current project version for new users.

Many other projects have it this way, for example:
* [Spring Framework](http://projects.spring.io/spring-framework/)
* [HikariCP](https://github.com/brettwooldridge/HikariCP)

The downside is indeed that with every release, the version number has to be updated in the README too.